libs/core/src/lib/toggle/toggle.component.ts

```diff
@@ -42,12 +42,13 @@ export class ToggleComponent implements OnInit, ControlValueAccessor {
     @ViewChild('input', { static: false })
     inputElement: ElementRef<HTMLInputElement>;
 
-    /**
-     * The size of the toggle.
-     * Can be one of the four *xs*, *s*, *l*, *error* or default.
-     */
+    /** If the toggle should have text in it or not. */
+    @Input()
+    noText: boolean = true;
+
+    /** For the toggle's direction. */
     @Input()
-    size: string;
+    dir: string = 'left';
```

Comment:
We don't need dir there. It is set on host element and will work same as for other components